requirements.txt

```diff
@@ -9,6 +9,8 @@ periodictable
 
 # bridges
 biopython==1.76
+openbabel==2.4.1
```

Comment:
Our setup is super confusing right now, but the Open Babel line actually needs to be removed; it's what's causing CI to fail.

Reply:
It eludes me what this is the case, but for the purposes of this PR we can submit once you remove the line and it fixes Travis.

Reply:
Right, removed.